boards/unphone8.json

```diff
@@ -0,0 +1,41 @@
+{
+  "build": {
+    "arduino":{
+      "ldscript": "esp32s3_out.ld"
+    },
+    "core": "esp32",
+    "extra_flags": [
+      "-DUNPHONE_SPIN=8",
+      "-DARDUINO_USB_CDC_ON_BOOT=0",
+      "-DARDUINO_RUNNING_CORE=1",
+      "-DARDUINO_EVENT_RUNNING_CORE=1"
+    ],
+    "f_cpu": "240000000L",
+    "f_flash": "80000000L",
+    "flash_mode": "dio",
+    "mcu": "esp32s3",
+    "variant": "unphone8"
+  },
+  "connectivity": [
+    "wifi"
+  ],
+  "debug": {
+    "openocd_board": "esp32s3.cfg"
+  },
+  "frameworks": [
+    "arduino",
+    "espidf"
+  ],
+  "name": "unPhone 8",
+  "upload": {
+    "flash_size": "8MB",
+    "maximum_ram_size": 327680,
+    "maximum_size": 4194304,
```

Comment:
I guess it should be `8388608` as you specified 8MB in the `flash_size` field.

Reply:
Good point, will change!